nfts/src/tests/extrinsics.rs

```diff
@@ -9,375 +9,665 @@ fn origin(account: u64) -> mock::Origin {
 	RawOrigin::Signed(account).into()
 }
 
-/* fn root() -> mock::Origin {
+fn root() -> mock::Origin {
 	RawOrigin::Root.into()
-} */
+}
 
-mod transfer {
+mod create {
 	use super::*;
 
 	#[test]
-	fn cannot_transfer_delegated_nfts() {
-		ExtBuilder::new_build(vec![(ALICE, 100)]).execute_with(|| {
-			let nft_id = NFTs::create_nft(ALICE, vec![0], None).unwrap();
-			assert_ok!(NFTs::set_viewer(nft_id, Some(BOB)));
+	fn create_ok_with_no_series() {
```

Comment:
You should have two happy tests.

One with series id set to None and one with a series id set to a new value.

Reply:
Also, the test name should not contain the word "ok" in them.

Reply:
Done